app/models/util/index.coffee

```diff
@@ -18,4 +20,8 @@ module.exports.getConfigPath = () ->
 
 module.exports.getConfigModulePath = () ->
   return null if ! (configPath = @getConfigPath())? || ! appConfig.defaultNamespace
-  return jetpack.cwd(configPath).path appConfig.defaultNamespace
\ No newline at end of file
+  return jetpack.cwd(configPath).path appConfig.defaultNamespace
+
+module.exports.openExternalUrl = (url) ->
+  return false if ! url?
```

Comment:
fixed in https://github.com/mazehall/eintopf/commit/ea60538807b3d06535f69881d62fad894fc5b0e9